modules/op-chains-generating-cosign-secret.adoc

```diff
@@ -0,0 +1,70 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="chains-generating-cosign-secret_{context}"]
+= Generating the cosign key pair by using the TektonConfig CR
+
+To use the `cosign` signing scheme for {tekton-chains} secrets, you can generate a `cosign` key pair that uses ECDSA encryption by setting the `generateSigningSecret` field in the `TektonConfig` custom resource (CR) to `true`.
+
+.Prerequisites
+
+* You installed the {oc-first} utility.
+* You are logged in to your {OCP} cluster with administrative rights for the `openshift-pipelines` namespace.
+
+.Procedure
+
+. Edit the `TektonConfig` CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit TektonConfig config
+----
+
+. In the `TektonConfig` CR, set the `generateSigningSecret` value to `true`:
++
+.Example of creating an ECDSA cosign key pair by using the TektonConfig CR
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  chain:
+    disabled: false
+    generateSigningSecret: true # <1>
+# ...
+----
+<1> The default value is `false`. Setting the value to `true` generates the `ecdsa` key pair.
+
+. After a few minutes, extract the public key from the secret and store it, so that you can use it to verify artifact attestations. Run the following command to extract the key:
```

Comment:
Will the phrase "artifact attestations" be familiar to the intended audience of this doc? I don't see similar phrasing when I search the openshift-docs repo.

Reply:
It will. https://docs.github.com/en/actions/security-for-github-actions/using-artifact-attestations/using-artifact-attestations-to-establish-provenance-for-builds